model/src/id.rs

```diff
@@ -759,6 +759,56 @@ impl From<NonZeroU64> for StageId {
     }
 }
 
+/// Unique ID of a sticker.
+#[allow(clippy::unsafe_derive_deserialize)]
+#[derive(Clone, Copy, Debug, Deserialize, Eq, Hash, Ord, PartialEq, PartialOrd, Serialize)]
+pub struct StickerId(#[serde(with = "string")] pub NonZeroU64);
+
+impl StickerId {
```

Comment:
we should remove this for now, and in a separate pr, we can deprecate the old sticker related ids